src/librustdoc/json/mod.rs

```diff
@@ -171,8 +171,21 @@ impl<'tcx> FormatRenderer<'tcx> for JsonRenderer<'tcx> {
     /// the hashmap because certain items (traits and types) need to have their mappings for trait
     /// implementations filled out before they're inserted.
     fn item(&mut self, item: clean::Item) -> Result<(), Error> {
+        let local_blanket_impl = match item.def_id {
+            clean::ItemId::Blanket { impl_id, .. } => impl_id.is_local(),
+            clean::ItemId::Auto { .. }
+            | clean::ItemId::DefId(_)
+            | clean::ItemId::Primitive(_, _) => false,
```

Comment:
Why not match these all with `_ => false`?

Reply:
When it was inside the ItemId def, it matched other methods. I figured it didn't hurt to ensure future changes required a decision on this code, as a future type may or may not want to do the same child-skipping trick until the underlying issues is fixed.